testing/iprutils/APKBUILD

```diff
@@ -0,0 +1,42 @@
+# Maintainer: Breno Leitao <breno.leitao@gmail.com>
+pkgname=iprutils
+pkgver=2.4.14
+pkgrel=0
+pkgdesc="user space tools required by the ipr device driver."
+url="https://github.com/bjking1/iprutils"
+arch="all"
```

Comment:
Even if this builds on all, is it useful? Or would it make sense to be limited to ppc?